### PR TITLE
Add support of the Holding event

### PR DESCRIPTION
**Pull request by dr1rrb**
GitHub Issue: https://github.com/unoplatform/uno/issues/2423

## Feature
Add support of the `UIElement.Holding` event.

## What is the current behavior?
`UIElement.Holding` event is never raised

## What is the new behavior?
`UIElement.Holding` event is raised

## PR Checklist
- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Holding_Tests.cs [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+			var result = GestureResult.Get(_app.Marked("LastHeld"));
+			result.Element.Should().Be(targetName);
+			result.State.Should().Be("Completed");
+			((int)result.X).Should().Be(tapX);
+			((int)result.Y).Should().Be(tapY);
+		}
+
+		[Test]
+		[AutoRetry]
+		[Ignore("Currently there is no available gesture to test cancel")]
```
Comment (jeromelaban) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Holding_Tests.cs]: Issue to track ?

Comment (dr1rrb) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/Holding_Tests.cs]: https://github.com/unoplatform/Uno.UITest/issues/31

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/GestureResult.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@ public static GestureResult Get(QueryEx textBlock)
 
 		public static GestureResult Parse(string text)
 		{
-			var regex = new Regex(@"(?<elt>[\w_]+)@(?<x>[\d\.]+),(?<y>[\d\.]+)");
+			var regex = new Regex(@"(?<elt>[\w_]+)(-(?<state>\w+))?@(?<x>[\d\.]+),(?<y>[\d\.]+)");
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/GestureResult.cs]: Regex are expensives. Maybe you should parse the string _more manually_.

File: src/Uno.UWP/System/DispatcherQueueTimer.cs [hunk 3]
```diff
@@ -0,0 +1,128 @@
+using System;
+using System.Threading;
+using Windows.Foundation;
+
+namespace Windows.System
+{
+	public partial class DispatcherQueueTimer
```
Comment (carldebilly) [inline src/Uno.UWP/System/DispatcherQueueTimer.cs]: This class should have a destructor calling the `.Stop()` method.
+ partial declarations missing.

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/GestureResult.cs (hunk 2 above)
Comment (dr1rrb) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Input/GestureResult.cs]: It's for UI tests

File: src/Uno.UWP/System/DispatcherQueueTimer.cs (hunk 3 above)
Comment (dr1rrb) [reply src/Uno.UWP/System/DispatcherQueueTimer.cs]: Good point for the finalizer, added !

`partial` are irrelevant here, they are just causing silent errors.

**Pull request merged**